extensions/MacOSX/kDriveLiteSync/Extension/xpcService.m

```diff
@@ -116,6 +116,7 @@ - (void)initOpenBlackListSet
                             @"com.apple.security.codesign",                     // codesign
                             @"com.apple.mdmclient",                             // MDM
                             @"com.apple.warmd_agent",                           // Warmd
+                            @"com.apple.apfsd",                                 // Apfsd
```

Comment:
Is there a document intended for users that lists all common programs which interfere with kDrive's `LiteSyncExtension`?

Reply:
No